packages/@okta/vuepress-site/docs/guides/configure-amr-claims-mapping/main/index.md

```diff
@@ -18,7 +18,7 @@ This guide explains how to configure an OpenID Connect Identity Provider to send
 #### What you need
 
 * [Okta Developer Edition organization](https://developer.okta.com/signup)
-* An existing OpenID Connect Identity Provider (IdP) that's able to send AMR claims to Okta. This can be another Okta org (org2org) or a third party IdP.
+* An existing OpenID Connect Identity Provider (IdP) that's able to send AMR claims to Okta. This can be another Okta org (Org2Org) or a third party IdP. If your current Org2Org uses SAML, you need to implement a new one with OpenID Connect.
```

Comment:
```suggestion
* An existing OpenID Connect Identity Provider (IdP) that's able to send AMR claims to Okta. This can be another Okta org (Org2Org) or a third party IdP. If your current Org2Org integration uses SAML, you need to implement a new one with OpenID Connect.
```